resources/assets/js/app/views/topics/topicCreate.js

```diff
@@ -0,0 +1,52 @@
+var Backbone = require('backbone');
+var Marionette = require('backbone.marionette');
+var TopicModel = require('../../Models/TopicModel');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'topicCreateNew',
+    model: new TopicModel(),
```

Comment:
don't specify instance like this, model should be passed on action level and not in module define